x-pack/plugins/security_solution/common/api/detection_engine/signals/index.ts

```diff
@@ -5,6 +5,11 @@
  * 2.0.
  */
 
+export * from './create_index/create_index_route';
+export * from './delete_index/delete_index_route';
 export * from './query_signals/query_signals_route';
+export * from './read_alerts_index_exists/read_alerts_index_exists_route';
+export * from './read_index/read_index_route';
+export * from './read_privileges/read_privileges_route';
```

Comment:
why index related route schemas moved to signals domain? 
URLs of these routes do not contain `signals` section. Something like `index` or `index_management` might be more suitable.